### PR TITLE
feat(route): add 健康界

### PR DESCRIPTION
**Pull request by qnloft**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/cn-healthcare/index
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [x] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [x] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

## 说明 / Note

健康界-RSS，网站地址：https://www.cn-healthcare.com/

### CODE REVIEW

**Review by TonyRL — CHANGES_REQUESTED** [review]
New route(s) should follow [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html) which will not have any changes to `assets/radar-rules.js` and `lib/router.js`

**Comment by qnloft** [comment]
> New route(s) should follow [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html) which will not have any changes to `assets/radar-rules.js` and `lib/router.js`

已修改

File: lib/v2/healthcare/index.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        // 源说明
+        description: `${name} - RSS`,
+        // 遍历此前获取的数据
+        item: data.datalist.map((item) => ({
+            // 文章标题
+            title: item.title,
+            // 文章正文
+            description: String(item.content),
+            // 文章发布时间
+            pubDate: new Date(item.createdate).toUTCString(),
```
Comment (TonyRL) [inline lib/v2/healthcare/index.js:25]:
```suggestion
    const name = '健康界 [cn-healthcare]';
    ctx.state.data = {
        // 源标题
        title: name,
        // 源链接
        link: 'https://www.cn-healthcare.com',
        // 源说明
        description: `${name} - RSS`,
        // 遍历此前获取的数据
        item: data.datalist.map((item) => ({
            // 文章标题
            title: item.title,
            // 文章正文
            description: item.content,
            // 文章发布时间
            pubDate: parseDate(item.createdate),
```
docs: https://docs.rsshub.app/joinus/pub-date.html#shi-yong-gong-ju-lei-parse-date

**Review by TonyRL — COMMENTED** [review]
Please remove your changes on `assets/radar-rules.js` and `lib/router.js` as it is already done in v2 routes.
I would suggest using the folder name `cn-healthcare` instead of `healthcare` to match the domain name.

File: lib/v2/cn-healthcare/index.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        // 源链接
+        link: 'https://www.cn-healthcare.com',
+        // 源说明
+        description: `${name} - RSS`,
+        // 遍历此前获取的数据
+        item: data.datalist.map((item) => ({
+            // 文章标题
+            title: item.title,
+            // 文章正文
+            description: String(item.content),
```
Comment (TonyRL) [inline lib/v2/cn-healthcare/index.js:24]:
```suggestion
            description: item.content,
```

**Review by TonyRL — COMMENTED** [review]
`lib/routes/healthcare/index.js` should also be removed as it's not being used.

File: docs/new-media.md [hunk 3]
```diff
@@ -4062,4 +4062,10 @@ QueryString:
 | ---- | ----- | ----- |
 |      | zh-hk | zh-tw |
 
+## 健康界
+
+### 首页
+
+<Route author="qnloft" example="/healthcare/index" path="/healthcare/index" />
```
Comment (TonyRL) [inline docs/new-media.md:4069]:
```suggestion
<Route author="qnloft" example="/cn-healthcare/index" path="/cn-healthcare/index" />
```

File: docs/new-media.md [hunk 4]
```diff
@@ -4061,5 +4069,3 @@ QueryString:
 | 简体中文 | 港澳繁體  | 台灣正體  |
 | ---- | ----- | ----- |
 |      | zh-hk | zh-tw |
```
Comment (TonyRL) [inline docs/new-media.md:4071]:
```suggestion
|      | zh-hk | zh-tw |

 </Route>
```

**Pull request merged**